pkgs/common-updater/scripts/list-archive-versions

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+set -x
```

Comment:
Not sure if we want to set this all the time.

Reply:
Done.